File: ssaExample/types_environment.java
import java.util.Map;

public class types_environment {

	public types_declInfo decl;
	public types_Scope scope;
	public int pos;
	public Object iota;
	public Object errpos;
	public Boolean inTParamList;
	public types_Signature sig;
	public Map<ast_CallExpr, Boolean> isPanic;
	public Boolean hasLabel;
	public Boolean hasCallOrRecv;
}

File: ssaExample/types_environment.kt
class types_environment {

	var decl: types_declInfo? = null
	var scope: types_Scope? = null
	var pos: Int? = null
	var iota: Any? = null
	var errpos: Any? = null
	var inTParamList: Boolean? = null
	var sig: types_Signature? = null
	var isPanic: Map<ast_CallExpr, Boolean>? = null
	var hasLabel: Boolean? = null
	var hasCallOrRecv: Boolean? = null
}
